test/manager/cdn/_fixtures/sample.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <script type="text/javascript"
```

Comment:
Some other suggestions for test cases:
1. relative protocol i.e. `//cdnjs.cloudflare.com`
2. HTTP sample
3. An invalid script tag i.e. `<script src="..." />`. This is invalid, but browsers will accept it.
4. CSS example for cdnjs